tls/s2n_tls13_handshake.c

```diff
@@ -286,6 +286,11 @@ int s2n_tls13_handle_handshake_traffic_secret(struct s2n_connection *conn, s2n_m
         session_key = &conn->secure.client_key;
         secret_type = S2N_CLIENT_HANDSHAKE_TRAFFIC_SECRET;
         conn->client = &conn->secure;
+
+        /* The client session key might have been set to the early traffic key.
+         * This is not an problem for most libcrypto implementations, but in older
+         * versions of OpenSSL will cause a memory leak when we call init again. */
```

Comment:
Can we specifically call out the older versions of OpenSSL and remove the ambiguity, maybe  say `in OpenSSL versions <=  1.0.2` instead of `in older versions of OpenSSL`?